<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>589</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#periodFrame{ border-style: outset; border-width: 2px; border-color: PaleTurquoise}
QFrame#scheduleFrame{ border-style: outset; border-width: 2px; border-color: PaleTurquoise}
QFrame#resultFrame{ border-style: outset; border-width: 2px; border-color: PaleTurquoise}</string>
  </property>
  <layout class="QVBoxLayout" name="dLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>150</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>762</width>
        <height>563</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="statusCheckbox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ruleLineEdit">
         <property name="placeholderText">
          <string>Rule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="formWidget" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="formWidgetLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="scheduleWidgetsLayout">
            <item>
             <widget class="QFrame" name="periodFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Duration</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="periodLabel">
                   <property name="text">
                    <string>Period(min)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="periodSpinBox">
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="value">
                    <number>15</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="scheduleFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="sfLayout">
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Schedule</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="dateLayout">
                 <item>
                  <widget class="QLabel" name="fromLable">
                   <property name="text">
                    <string>From Date:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="fromDate">
                   <property name="displayFormat">
                    <string>yyyy/MM/dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="toLabel">
                   <property name="text">
                    <string>To Date:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="toDate">
                   <property name="displayFormat">
                    <string>yyyy/MM/dd</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="timeLayout">
                 <item>
                  <widget class="QLabel" name="fromTLabel">
                   <property name="text">
                    <string>From Time:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTimeEdit" name="fromTime"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="toTLabel">
                   <property name="text">
                    <string>To Time:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTimeEdit" name="toTime">
                   <property name="dateTime">
                    <datetime>
                     <hour>23</hour>
                     <minute>59</minute>
                     <second>59</second>
                     <year>2000</year>
                     <month>1</month>
                     <day>1</day>
                    </datetime>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="resultFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Result</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="resultCombo"/>
               </item>
               <item>
                <widget class="QLineEdit" name="resultLineEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
